SignalShareExtension/ShareViewController.swift

```diff
@@ -249,7 +249,10 @@ public class ShareViewController: UINavigationController, ShareViewDelegate, SAE
             // is has already saved their local profile key in the main app.
             showNotReadyView()
         } else {
-            presentConversationPicker()
+            DispatchQueue.main.async { [weak self] in
```

Comment:
`presentConversationPicker` can be expensive, so I've modified this to call it async (still on `main`).  That should help ensure that loading the share view controller occurs in a timely way.